examples/spear_qcp/scenario.txt

```diff
@@ -1,4 +1,4 @@
-algo = python -u ./target_algorithm/scripts/SATCSSCWrapper.py --mem-limit 1024 --script ./target_algorithm/spear-python/spearCSSCWrapper.py 
+algo = python2 -u ./target_algorithm/scripts/SATCSSCWrapper.py --mem-limit 1024 --script ./target_algorithm/spear-python/spearCSSCWrapper.py
```

Comment:
Is there a reason to specifically use `python2` here?

Reply:
This prevents the python2 script SATCSSCWrapper.py from being run with python3 when python3 is the default, i.e. on Linux /usr/bin/python is a symlink to python3. As no features that are incompatible between python2 and python3 are used in the script, this original code magically works but in the general case it does not. This is the reason I explicitly call python2.

Reply:
If the script works in python3 as well, I'd rather change the script to `python3` to make us more forward-compatible.

Reply:
This means I also have to change the shebang line in SATCSCWrapper.py. I'm doing this without explicitly reviewing the code. I consider this bad practice (the reason I changed it to explicitly use python2 and not python3) but for this case I'm doing it anyway.

Reply:
@mlindauer any issues here?

Reply:
There is no shebang in SATCSSCWrapper.py. So, I don't understand it.

in general, we don't support Python 2 and thus, all our examples should be based on Python 3